mozilla/bedrock
# Dependency updates

## One-line summary

Dependency updates

#12338 Bump boto3 from 1.24.96 to 1.26.3 in /requirements
#12334 Bump twilio from 7.14.0 to 7.15.1 in /requirements
#12333 Bump phonenumberslite from 8.12.57 to 8.13.0 in /requirements
#12312 Bump everett from 3.0.0 to 3.1.0 in /requirements
#12309 Bump pillow from 9.2.0 to 9.3.0 in /requirements
#12306 Bump newrelic from 8.2.1 to 8.4.0 in /requirements

PLUS pin requests to 2.28 to get rid of warning:
> RequestsDependencyWarning: urllib3 (1.26.11) or
> chardet (5.0.0)/charset_normalizer (2.0.12) doesn't
> match a supported version!

### TO TEST
Please check this out, reinstall all deps and run `pytest bedrock` because it was hanging for me locally and I don't want that to be happening for anyone else. (The tests also hang for me on `main`, so I don't think it's a regression, but I want to be sure, obvs)
